ls1intum/Artemis
# Feature/grading instructions/display sgi in assessment view fix

### Checklist

### Motivation and Context


As multiple tutors asses student's submission, the assessment becomes inconsistent.
Tutors used to insert free text feedback and apply their individual interpretation of the grading instructions. This leads to the feedback being sometimes faulty or incomplete. Therefore, instructors can predefine feedback text that tutors can directly drag and drop. This also saves tutors time as they don't need to type the same feedback over and over again.
### Description

The structured grading instructions are displayed at three different Places:
a.) At the exercise dashboard before Tutors participate in the assessment
b.) At the side panel of the exercise dashboard after tutors submit that they read the instructions
c.) beside students submission in the tutor assessment view
The instructions could be dragged and dropped to the feedback element.
The instructions div color depends on the score: 
< 0 : red
=0 : yellow
>0 : green

### Steps for Testing

make sure that you are signed in as an admin:
1. Create a file-upload or text exercise with structured grading instructions
2. Create a student's submission for the exercise you created
3. Navigate to the tutor dashboard -> the structured grading instructions should be displayed there
4. Start assessing the submission
5. Drag and drop an instruction to the feedback area -> the score and feedback text should be applied
6. Drag and drop an instruction to the general feedback field as well


### Screenshots


a.)
![image](https://user-images.githubusercontent.com/50635005/77358960-25a88f80-6d4b-11ea-8872-6143a5ff3341.png)
b.)
![image](https://user-images.githubusercontent.com/50635005/77359009-3d801380-6d4b-11ea-973a-e24ad15e9e54.png)
c.)
![image](https://user-images.githubusercontent.com/50635005/77359206-833cdc00-6d4b-11ea-9b09-30dd34ac671e.png)
Drag & Drop to feedback element
![image](https://user-images.githubusercontent.com/50635005/77359249-96e84280-6d4b-11ea-8dd1-afaf63dc33a4.png)
![image](https://user-images.githubusercontent.com/50635005/77359284-a4053180-6d4b-11ea-8473-b485e3fc3bb5.png)
Drag & Drop to general feedback field
![image](https://user-images.githubusercontent.com/50635005/77359353-ba12f200-6d4b-11ea-9797-0d6a0cef3655.png)
![image](https://user-images.githubusercontent.com/50635005/77359380-c5feb400-6d4b-11ea-9497-3e7234e5ec60.png)

If an element is missing in the text exercise submission tutors can drop the instruction to the general feedback field but in a separate PR this should be addressed by enabling tutors to create a proper feedback element.
